lib/ui/platform_dispatcher.dart

```diff
@@ -308,6 +308,15 @@ class PlatformDispatcher {
     _invoke(onMetricsChanged, _onMetricsChangedZone);
   }
 
+  // [FlutterView]s for which [FlutterView.render] has already been called
+  // during the current [onBeginFrame]/[onDrawFrame] callback sequence.
+  //
+  // The field is null outside the scope of those callbacks indicating that
+  // calls to [FlutterView.render] must be ignored. Furthermore, if a given
+  // [FlutterView] is already present in this set when its [FlutterView.render]
+  // is called again, that call must be ignored as a duplicate.
+  Set<FlutterView>? _renderedViews;
```

Comment:
```suggestion
  // [FlutterView]s for which [FlutterView.render] has already been called
  // during the current `_beginFrame`/`_drawFrame` callback sequence.
  //
  // The field is null outside the scope of those callbacks; specifically, its value is
  // temporarily stored in `_renderedViewsInPastCalls` at the end of `_beginFrame`
  // so that the value can survive through the gap. In this way,
  // calls to [FlutterView.render] out of those callbacks are ignored, and
  // one view can only be rendered once in a callback sequence.
  Set<FlutterView>? _renderedViews;
```